ui/src/components/table/CustomTable.tsx

```diff
@@ -213,8 +242,8 @@ function CustomTable({
                     columns.map((headData) => (
                         <Table.HeadCell
                             key={headData.field}
-                            onSort={headData.sortKey ? handleSort : null}
-                            sortKey={headData.sortKey ? headData.sortKey : null}
+                            onSort={headData.sortKey ? handleSort : undefined}
+                            sortKey={headData.sortKey || ''}
```

Comment:
it accepts string or undefined

```suggestion
                            sortKey={headData.sortKey ?? undefined }
```

Reply:
or even better if we can have sortKey optional during the column mapping

Reply:
done